MANIFEST.in

```diff
@@ -1,5 +1,15 @@
+include CONTRIBUTING.rst
+include LICENSE
+include README.md
+include caproto/_version.py
+include caproto/server/*.jinja2
+include docs-requirements.txt
 include generate_headers.py
 include headers.tpl
+include test-requirements.txt
 include versioneer.py
-include caproto/_version.py
-include caproto/server/*.jinja2
+
+recursive-exclude * __pycache__
+recursive-exclude * *.py[co]
+
+recursive-include docs *.rst *.py Makefile make.bat
```

Comment:
I think `docs` was meant to be `doc`? Do we want that to be included in a source install?

Reply:
I followed the pattern from https://github.com/NSLS-II/scientific-python-cookiecutter/blob/master/%7B%7B%20cookiecutter.repo_name%20%7D%7D/MANIFEST.in, so included the docs.

Reply:
I think the comment is about how the directory is spelled in this repo, not questioning whether to include it.

Reply:
It was both! 

Need to drop the `s` from `docs` for it to work. 

But also: Is there value in shipping the sphinx docs source? I see other major projects also doing it, so the answer is probably yes. But I can't think of how / when it would ever be used.

Reply:
OK, one more fix is coming :)
However, @MikeHart85's second question was about including it or skip the docs from the distribution. I think it's a good practice to also supply the docs, so that they can be built off the published version.

Reply:
A fix is pushed. It's a bit confusing to have the `doc` dir, but `docs-requirements.txt`. Maybe this can be revised to also follow our decision to have a pattern like `requirements-<>.txt` as it's easier to tab-complete and look in the sorted version.

Reply:
Sounds good to me.

Reply:
OK, I renamed them in my latest commit: https://github.com/caproto/caproto/pull/611/commits/21943a6a2a4b1857e34a339187f470f5ceaf3467.